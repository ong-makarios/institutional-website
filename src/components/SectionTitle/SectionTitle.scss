@import "../../styles/helpers.scss";

.section-title {
  margin-top: 40px;
  border-bottom: 2px solid $white-smoke;
  padding-bottom: 15px;
}
