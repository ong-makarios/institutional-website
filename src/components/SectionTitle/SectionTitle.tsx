import React from "react";
import "./SectionTitle.scss";
import { SectionTitleProps } from "../../types/props";

const SectionTitle: React.FC<SectionTitleProps> = ({ title }) => {
  return <h3 className="section-title">{title}</h3>;
};

export default SectionTitle;
