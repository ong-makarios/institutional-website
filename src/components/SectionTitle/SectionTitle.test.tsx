import React from "react";
import { render, screen } from "@testing-library/react";
import SectionTitle from "./SectionTitle";

describe("SectionTitle", () => {
  it("should render correctly", () => {
    const { container } = render(<SectionTitle title="Test" />);

    expect(container).toMatchSnapshot();
  });

  it("should render title passed in props", () => {
    render(<SectionTitle title="Test" />);

    expect(screen.getByText("Test")).toBeInTheDocument();
  });
});
