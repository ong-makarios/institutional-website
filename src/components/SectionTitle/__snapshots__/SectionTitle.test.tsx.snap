// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle should render correctly 1`] = `
<div>
  <h3
    class="section-title"
  >
    Test
  </h3>
</div>
`;
